widget/embedded/src/components/Route/Route.tsx

```diff
@@ -0,0 +1,266 @@
+                        </Typography>
+                        <Divider direction="horizontal" size={8} />
+                        <Typography
+                          size="xsmall"
+                          variant="body"
+                          color="neutral900">
+                          |
+                        </Typography>
+                        <Divider direction="horizontal" size={8} />
+                        <Typography
+                          size="xsmall"
+                          variant="body"
+                          color="neutral900">
+                          {i18n.t({
+                            id: 'yourSymbol',
+                            message: 'Yours: {amount} {symbol}',
+                            values: {
+                              amount: numberToString(
+                                swap.fromAmount,
+                                TOKEN_AMOUNT_MIN_DECIMALS,
+                                TOKEN_AMOUNT_MAX_DECIMALS
+                              ),
+                              symbol: swap?.from.symbol,
+                            },
+                          })}
+                        </Typography>
+                      </>
+                    )}
+                    {(error?.type === RouteErrorType.INSUFFICIENT_SLIPPAGE ||
+                      warning?.type ===
+                        RouteWarningType.INSUFFICIENT_SLIPPAGE) && (
+                      <Typography
+                        size="xsmall"
+                        variant="body"
+                        color="neutral900">
+                        {i18n.t({
+                          id: 'minRequiredSlippagePerStep',
+                          message:
+                            'min required Slippage: {minRequiredSlippage}',
+                          values: {
+                            ...(error?.type ===
+                              RouteErrorType.INSUFFICIENT_SLIPPAGE && {
+                              minRequiredSlippage:
+                                error.recommendedSlippages?.get(index),
+                            }),
+                            ...(warning?.type ===
+                              RouteWarningType.INSUFFICIENT_SLIPPAGE && {
+                              minRequiredSlippage:
+                                warning.recommendedSlippages?.get(index),
+                            }),
+                          },
+                        })}
+                      </Typography>
+                    )}
+                  </FooterAlert>
+                }
+              />
+            </FooterStepAlarm>
+          ) : undefined,
+      };
+    });
+  };
+
+  const showBestRoute = bestRoute && bestRoute.result && !loading;
+  return (
+    <div>
+      {loading && (
+        <BestRouteContainer>
+          <BestRouteSkeleton type={type} expanded={expanded} />
+        </BestRouteContainer>
+      )}
+      {showBestRoute ? (
+        <BestRouteContainer>
+          <BestRoute
```

Comment:
I think we should improve the `BestRoute` props a bit here. Instead of passing the values as separate props (e.g. `totalFee`, `totalTime` and `steps`) which all of them are reading from `bestRoute` object, we can pass the whole object in a single prop (like `swap`/`route`) and do the calculation inside the `BestRoute` component itself (move `getBestRouteSteps` into the component).